components/banners/HeroBanner.vue

```diff
@@ -27,27 +33,46 @@
       subtitle: String,
       desktopUrl: String,
       mobileUrl: String,
-      imageAlt: String
+      imageAlt: String,
     }
   }
+
+
+
+
 </script>
 
-<style scoped lang="scss">
+<style lang="scss">
   .hero-banner {
```

Comment:
it's not scoped here so the child `BannerContent` component can inherit styles. I don't know if that is right.